internal/pkg/policy/policy_output.go

```diff
@@ -35,9 +38,10 @@ var (
 )
 
 type Output struct {
-	Name string
-	Type string
-	Role *RoleT
+	Name         string
+	Type         string
+	ServiceToken string `json:"service_token,omitempty"`
```

Comment:
Any reason for having json tag on'y for this one?

Reply:
the Name and Type field has the same name in the json, so I though only need to map ServiceToken which is different. I can add json tag for the others too for consistency